### ISSUE TITLE
remove bounds on `init_R` and `rw_sd`

### ISSUE THREAD

**Issue by sbfnk** [nfidd/nfidd#234]
Vague memory is that this is in another issue somewhere but I couldn't find it - if we set the `init` we shouldn't need this (and experimentation in the Rt session confirms we don't).

**Comment by seabbs** [comment]
this is blocked by. #207

**Comment by sbfnk** [comment]
Can I remove them elsewhere?
